paperqa/agents/search.py

```diff
@@ -429,35 +429,36 @@ async def query(
 
 def fetch_kwargs_from_manifest(
     file_location: str, manifest: dict[str, Any], manifest_fallback_location: str
-) -> dict[str, Any]:
-    manifest_entry: DocDetails | None = manifest.get(file_location) or manifest.get(
+) -> dict[Any, Any]:
```

Comment:
Why do we need to do this? `dict[str, Any]` should be compatible with `model_dump`